test/gtest/uct/test_p2p_am.cc

```diff
@@ -150,7 +150,7 @@ class uct_p2p_am_test : public uct_p2p_test
             short_progress_loop();
         }
 
-        status = uct_iface_set_am_handler(receiver().iface(), AM_ID, NULL, NULL);
+        status = uct_iface_set_am_handler(receiver().iface(), AM_ID, NULL, NULL, 0);
```

Comment:
Yep. @alex-mikheev maybe we could add it as another dimension to uct_p2p_am test?